src/Symfony/Component/AssetMapper/AssetMapper.php

```diff
@@ -239,11 +245,7 @@ private function getMimeType(string $logicalPath): ?string
 
         $extension = pathinfo($logicalPath, \PATHINFO_EXTENSION);
 
-        if (!isset($this->extensionsMap[$extension])) {
-            throw new \LogicException(sprintf('The file extension "%s" from "%s" does not correspond to any known types in the asset mapper. To support this extension, configure framework.asset_mapper.extensions.', $extension, $logicalPath));
-        }
-
-        return $this->extensionsMap[$extension];
+        return $this->extensionsMap[$extension] ?? null;
```

Comment:
This is unrelated. I had a file that mentioned a sourcemap - so `foo.js.map` - and the system exploded because we didn't know the mime type of `.map`. The mime type is only important for (A) the dev server to return that asset with the correct`Content-Type` and (B) for asset compilers, if they rely on the mime type. So I think this is a reasonable thing to do.